classes/PipelineRequest/README.md

```diff
@@ -65,3 +65,59 @@ The error list defaults to an empty array.
 A callback that gets triggered when a request is sent.
 It expects references to the resolve and reject Promise callback functions.
 This method can be overridden by a child class.
+
+## Testing - MockedPipelineRequest
+### How to mock this module?
+
+It's easy to test functions that are using PipelineRequest. The only thing which is important is that tested function must return an original `Promise` when called. For example:
```

Comment:
Grammar: ... that **the** tested function ...

Reply:
done